### PR TITLE
Task #8266: Apply MVC Structure

### PR DESCRIPTION
**Pull request by tanbtd-1747**
### Issue Link :link:
https://edu-redmine.framgia.vn/issues/8266

### Goals :soccer:
- Initialize the project with MVC Architecture
- Include SwiftLint

### CODE REVIEW

File: Money*/Money*/Application/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        return true
+    }
+
```
Comment (cuongvx-0845) [inline Money*/Money*/Application/AppDelegate.swift]: thừa dòng

File: Money*/Money*/Scenes/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
```
Comment (cuongvx-0845) [inline Money*/Money*/Scenes/ViewController.swift]: thừa dòng

**Review by nguyenck-1-1519 — APPROVED** [review]

File: Money*/.swiftlint.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+force_try:
+  severity: warning # explicitly. Give warning only for force casting
+# rules that have both warning and error levels, can set just the warning level
+# implicitly
+function_body_length:
+  warning: 50
+  error: 60
+
+line_length: 160
```
Comment (tgdong2296) [inline Money*/.swiftlint.yml]: line length 120 thôi anh nhé

Comment (tanbtd-1747) [reply Money*/.swiftlint.yml]: đã fix nhé

**Review by tgdong2296 — APPROVED** [review]

**Review by cuongvx-0845 — APPROVED** [review]

**Review by minhduc2511 — APPROVED** [review]

**Review by tienpx3010 — APPROVED** [review]

**Pull request merged**